package.json

```diff
@@ -23,6 +23,7 @@
     "gulp": "gulp"
   },
   "dependencies": {
+    "@alkemio/visualization": "^1.0.2",
```

Comment:
none of our packages start at 1.0, they are all pre 1.0 - so please reset the versioning to be 0.1.0